request_handler.py

```diff
@@ -124,11 +131,15 @@ def do_PUT(self):
         if resource == "users":
             update_user(id, post_body)
         if resource == "comments":
-            update_comment(id, post_body)
+            success = update_comment(id, post_body)
```

Comment:
Take this line out, the condition is unnecessary and you should just have line 127

Reply:
If I understand your suggestion correctly, I think I have fixed it